src/flask_httpauth.py

```diff
@@ -14,6 +14,7 @@
 from random import Random, SystemRandom
 from flask import request, make_response, session, g, Response, current_app
 from werkzeug.datastructures import Authorization
+from werkzeug.formparser import _chunk_iter
```

Comment:
The `_` prefix in this function name indicates that it is private and is not supposed to be used externally. I have been burned many times by using private functions from Werkzeug that are routinely renamed or relocated to different modules, so I don't consider this an acceptable solution.